database/migrations/2016_02_17_175810_create_competition_user_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCompetitionUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('competition_user', function (Blueprint $table) {
```

Comment:
pivot table! we need this and the `user_waiting_room` pivot table which I'm adding as well :)